libsolidity/ast/Types.cpp

```diff
@@ -110,6 +110,13 @@ util::Result<TypePointers> transformParametersToExternal(TypePointers const& _pa
 	return transformed;
 }
 
+string typesToString(TypePointers const& _types, bool _short)
+{
+	return '(' + util::joinHumanReadable(
+			_types | ranges::views::transform([&](auto const* _type) { return _type->toString(_short); })
+		) + ')';
```

Comment:
(Even though also discussed via chat)
This probably needs to be
```suggestion
	return '(' + util::joinHumanReadable(
			_types | ranges::views::transform([&](auto const* _type) { return _type->toString(_short); }),
			","
		) + ')';
```
in order for things to stay consistent with the old behaviour of ``toString``.